### PR TITLE
Update reprap_discount_full_graphic_lcd

### PR DESCRIPTION
**Pull request by V1EngineeringInc**
The full graphic lcd is reversed on all boards but the full rambo. This is the fix.
Also adding a screen timeout to return home after 3 minutes.

### CODE REVIEW

**Comment by V1EngineeringInc** [comment]
Do I need to enable the option as well as set it, or does setting it also enable it?

File: src/configs/accessories/reprap_discount_full_graphic_lcd [hunk 1]
```diff
@@ -1,7 +1,9 @@
 opt_enable \
-    REVERSE_ENCODER_DIRECTION \
     SDSUPPORT \
-    REPRAP_DISCOUNT_FULL_GRAPHIC_SMART_CONTROLLER
+    REPRAP_DISCOUNT_FULL_GRAPHIC_SMART_CONTROLLER \
+    LCD_TIMEOUT_TO_STATUS
+    
+opt_set LCD_TIMEOUT_TO_STATUS "180000"
```
Comment (jeffeb3) [inline src/configs/accessories/reprap_discount_full_graphic_lcd:6]: Setting it also enables it, IIRC. So you don't need to do both.

**Comment by jeffeb3** [comment]
I'm so glad you're testing this stuff. This is desperately needed.

**Comment by V1EngineeringInc** [comment]
I finally have my test bench up in the office next to my faster computer so compiling is not tortuously slow. I have a mini, full rambo and skr in the test lineup right now. My archim was sent in for repairs.

I want to get this sorted so we can start using it.

I have jerk updates next.

**Comment by jeffeb3** [comment]
This looks good.

**Comment by jeffeb3** [comment]
Do you have a list of things you try out when testing? It doesn't have to be a huge list of requirements, but just things like:

- LCD knob goes the right way
- Crown runs
- motors don't get over 50C after crown
- Endstops show triggered
- Z min shows triggered
- Homing X, Y, Z
- G1 Z20 F99999999
- Movement works without endstops attached

Stuff like that? I wouldn't shoot for an exhaustive list (the crown might be too much). I call it a "fool me once" test. Anything we've broke in the past goes on there, and we make sure we don't break it again.

**Comment by V1EngineeringInc** [comment]
I will start a list. If it compiles the first thing I always notice is the LCD rotation. So as you can tell my testing has not progressed very far. For my inexperience it is best if I keep my PR's small anyway.

I am not 100% wired up yet so I will work on that while doing some basic tests, take notes on what I am looking at/for, and keep submitting PR's.

**Comment by jeffeb3** [comment]
Sounds good. It can be really cumbersome to test everything on many builds. 

- We can just do some quick tests on the builds before they get PRed, editing and building from a zip. That's not too hard to get right in a PR.
- Then merge the PRs without testing, as long as a bigger test is planned soon. The PRs do have to build though.
- When we get several of these worked out, the main merge commit will get built. That is the build to do the "fool me once" test on. 
- We should be able to just make a release right then. It is possible for upstream (marlin) changes to change the behavior between the merge commit and the release. But if they are close together, it seems reasonable to skip the test on the release too. It's a balance between time and risk.

So if you're going to do a few small PRs, it makes sense to just test them enough to be confident you fixed what you are trying to fix. No need to test the endstops when PRing the encoder direction, for example.

**Comment by V1EngineeringInc** [comment]
Yeah. Currently I am testing my own edits, then submit the PR and let it build then double check the edit stuck or propagated like I expect. I have not figured out how to edit multiple files on github.com without submitting a pr then editing it. Hence the funky triple edit on this one.

Checked on two boards, merging this one. Next up is edit some mp3dp jerk values and submitting that, flash some printers and test it all day on real prints.

**Pull request merged**